ndmg/scripts/ndmg_demo-mrilab

```diff
@@ -1,46 +0,0 @@
-#!/usr/bin/env/python
```

Comment:
Why is there a "file changed" edit removing a file that doesn't exist in the upstream fork?